x-pack/plugins/apm/public/components/routing/onboarding/redirect.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { useApmPluginContext } from '../../../context/apm_plugin/use_apm_plugin_context';
+
+function TutorialRedirect() {
+  const {
+    config: { serverlessOnboarding },
+    core: {
+      application: { navigateToUrl },
+      http: { basePath },
+    },
+  } = useApmPluginContext();
+
+  if (serverlessOnboarding) {
+    navigateToUrl(basePath.prepend('/app/apm/onboarding'));
```

Comment:
Does this do a push or replace? Since this is a redirect it should be a replace.

Reply:
It does a history.push, i was a bit confused if we should do a push or replace as replace does a whole page reload. I will change it to replace by adding the `forceRedirect: true`

Reply:
i see the problem now. Good catch.
`forceRedirect` was doing a `window.location.assign(path);` which for sure we don't need.
The `navigateToUrl` only allows `push` or `assign`.

But we have `navigateToApp` which allows a replace. Let me test and update it

Reply:
@sqren It's fixed now.

Reply:
Great, thanks!